lib/sift/client.rb

```diff
@@ -51,13 +53,13 @@ def ok?
     end
 
 
-    # DEPRECIATED
+    # DEPRECATED
```

Comment:
Just for good measure, you could make this same correction in L57 and L63 as well?